src/Admin/Menu/MainMenuBuilder.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Admin\Menu;
+
+use App\Attribute\AsMenuBuilder;
+use Knp\Menu\FactoryInterface;
+use Knp\Menu\ItemInterface;
+use Netgen\Conversations\Participant\ParticipantProviderInterface;
+use Netgen\Conversations\Repository\ConversationRepository;
+
+#[AsMenuBuilder('createMenu', 'app.admin.main_menu')]
+final class MainMenuBuilder
```

Comment:
Wrong rebase here?

Reply:
Yes, rebase error (this file was moved to `src/Backoffice/Menu`)
It is fixed now.